libs/libskynet/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"compilerOptions": {
+		"target": "es2020",
```

Comment:
It looks pretty safe to use es2021 these days as all up to date browsers and node lts seems to support it.
http://kangax.github.io/compat-table/es2016plus/